appveyor.yml

```diff
@@ -57,7 +58,7 @@ install:
 build: false
 
 test_script:
-  - python -m pip wheel --wheel-dir=wheelhouse url || dir
+  - python -m pip wheel --wheel-dir=wheelhouse urlpy || dir
```

Comment:
Why not having this one wheel in the same list as above?

Reply:
brcause urlpy is not supported by python 37